IDL/constants.idl

```diff
@@ -0,0 +1,91 @@
+#include "helpers/basic_inner_types.idl"
+
+const short const_short = 1;
+const unsigned short const_ushort = 1;
+const long const_long = 1;
+const unsigned long const_ulong = 1;
+const long long const_longlong = 1;
+const unsigned long long const_ulonglong = 1;
+const float const_float = 1.1;
+const double const_double = 1.1;
+const long double const_longdouble = 1.1;
+const boolean const_boolean = true;
+const octet const_octet = 1;
+const char const_char8 = '1';
+const wchar const_char16 = L'1';
+const int8 const_int8 = 1;
+const uint8 const_uint8 = 1;
+const int16 const_int16 = 1;
+const uint16 const_uint16 = 1;
+const int32 const_int32 = 1;
+const uint32 const_uint32 = 1;
+const int64 const_int64 = 1;
+const uint64 const_uint64 = 1;
+const string const_string = "11";
+const wstring const_wstring = L"11";
+
+typedef short alias_short;
+const alias_short alias_const = 1;
+
+
+module const_module1
+{
+    const short const_moduled = 11;
+    typedef short alias_short_moduled;
+    const alias_short_moduled alias_const_moduled = 1;
+
+    struct ModuleConstsLiteralsStruct
+    {
+        short module1_array_literal_const_moduled[const_moduled];
+        short module1_array_literal_const_alias_const_moduled[alias_const_moduled];
+    };
+};
+
+module const_module2
+{
+    const short const_moduled = 22;
+    const const_module1::alias_short_moduled alias_const_moduled = 2;
+
+    struct Module2ConstsLiteralsStruct
+    {
+        short module2_array_literal_const_moduled[const_moduled];
+        short module2_array_literal_const_alias_const_moduled[alias_const_moduled];

```

Comment:
Please, add theses cases in this Module2ConstLiteralsStruct structure:

```omg-idl
        short module2_array_literal_const_scoped_moduled[const_module2::const_moduled];
        short module2_array_literal_module1_const_moduled[const_module1::const_moduled];
```

Reply:
added in [3656b48](https://github.com/eProsima/dds-types-test/pull/8/commits/3656b48bb0b974a4ad582a8ac93cb33cd0727ea7)